testsuite/features/allcli_reboot.feature

```diff
@@ -47,22 +47,31 @@ Feature: Reboot systems managed by SUSE Manager
     Then I wait and check that "sle-client" has rebooted
 
 @centos_minion
-  Scenario: Reboot the CentOS minion
+  Scenario: Reboot the CentOS minion and wait until reboot is completed
     Given I am on the Systems overview page of this "ceos-minion"
     When I follow first "Schedule System Reboot"
     Then I should see a "System Reboot Confirmation" text
     And I should see a "Reboot system" button
     When I click on "Reboot system"
     Then I should see a "Reboot scheduled for system" text
-
-@centos_minion
```

Comment:
I would keep the PR focused only for Ubuntu without touching other clients. Moreover, although the Ubuntu scenario looks simpler I'm not sure what was the initial reason for splitting reboot into two scenarios for the other clients. 
Let's add @Bischoff in the loop.

Reply:
Ups, sorry, missed the other comments.